web_src/css/repo.css

```diff
@@ -1225,6 +1225,11 @@
   height: 200px;
 }
 
+.milestone-progress-big {
+  width: min(420px, 96vw);
```

Comment:
Sorry but ... why "width: min" here?

* Does "420px" overflow some small width screens?
* Could the "96vw" be replaced by "flex: 1" ?

Reply:
I don't know how small screens can get, but `min` confines the bar neatly to the screen with equal margins. Could also use q media query to achieve the same effect without `vw` which is a bit cleaner.